rpxdock/fragments/extract_fragments.py

```diff
@@ -0,0 +1,255 @@
+import os, numpy as np, rpxdock as rp, rmsd
+
+def _reduce_dataset(dat):
+   return dat.drop([
+      't_fa_atr', 't_fa_rep', 't_fa_sol', 't_fa_intra_atr_xover4', 't_fa_intra_rep_xover4',
+      't_fa_intra_sol_xover4', 't_lk_ball', 't_lk_ball_iso', 't_lk_ball_bridge',
+      't_lk_ball_bridge_uncpl', 't_fa_elec', 't_fa_intra_elec', 't_pro_close', 't_hbond_sr_bb',
+      't_hbond_lr_bb', 't_hbond_bb_sc', 't_hb_sc', 't_dslf_fa13', 't_rama_prepro', 't_omega',
+      't_p_aa_pp', 't_fa_dun_rot', 't_fa_dun_dev', 't_fa_dun_semi', 't_hxl_tors', 't_ref',
+      'r_fa_sol', 'r_fa_intra_atr_xover4', 'r_fa_intra_rep_xover4', 'r_fa_intra_sol_xover4',
+      'r_lk_ball', 'r_lk_ball_iso', 'r_lk_ball_bridge', 'r_lk_ball_bridge_uncpl', 'r_fa_elec',
+      'r_fa_intra_elec', 'r_pro_close', 'r_hbond_sr_bb', 'r_hbond_lr_bb', 'r_hbond_bb_sc',
+      'r_hb_sc', 'r_dslf_fa13', 'r_rama_prepro', 'r_omega', 'r_p_aa_pp', 'r_fa_dun_rot',
+      'r_fa_dun_dev', 'r_fa_dun_semi', 'r_hxl_tors', 'r_ref', 'sasa2', 'sasa3', 'sasa4', 'nnb6',
+      'nnb8', 'nnb10', 'nnb12', 'nnb14', 'p_raw_etot', 'p_hb_bb_bb', 'p_hb_bb_sc', 'p_hb_sc_bb',
+      'p_hb_sc_sc', 'p_fa_sol', 'p_lk_ball', 'p_fa_elec'
+   ])
+
+def main():
+
+   kw = rp.Bunch(
+      min_helix1_len=14,
+      max_helix1_len=28,
+      min_turn_len=3,
+      max_turn_len=5,
+      min_helix2_len=14,
+      max_helix2_len=28,
+      max_e_per_res=-0.2,
+      max_bb_redundancy=2.0,
+   )
+
+   # fname = '/home/sheffler/data/respairdat/pdb_res_pair_data_si30_100.pickle'
+   fname = '/home/sheffler/data/respairdat/pdb_res_pair_data_si30.pickle'
+   datfname = fname.replace('/home/sheffler/data/respairdat/pdb_res_pair_data', 'tmp_hlh_dat')
+   boundsfname = fname.replace('/home/sheffler/data/respairdat/pdb_res_pair_data',
+                               'tmp_hlh_bounds')
+   hlhfname = fname.replace('/home/sheffler/data/respairdat/pdb_res_pair_data', 'tmp_hlh')
+
```

Comment:
these dependancies will not work anywhere else either. Is there additional db that we should be sharing beyond the ones that are currently in the outwards folder?